app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
   </head>
 
   <body>
+   <p class="notice"><%= notice %></p>
+       <p class="alert"><%= alert %></p>
     <%= yield %>
   </body>
```

Comment:
- [OPTIONAL] Although I highlighted codes in this file, I think you can implement the comment below in the `Users#Index` page. It would be nice to add a button that enables users to log out on the Users#Index page. It would improve the user's experience of your application. 😎